st2common/st2common/models/api/rule.py

```diff
@@ -162,7 +163,7 @@ def to_model(cls, rule):
 
         trigger = reference.get_str_resource_ref_from_model(trigger_db)
         criteria = dict(getattr(rule, 'criteria', {}))
-        pack = str(rule.pack)
+        pack = getattr(rule, 'pack', DEFAULT_PACK_NAME)
```

Comment:
+1 to what dzimine says. It should be consistent. We can either make `pack` required everywhere or optional everywhere. 

>Which was handy for creating them in one pack (e.g. default) and moving to another without changing the metadata files.

There is a problem that we don't clean up the one in default.